cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1265,3 +1265,28 @@ declare interface WorkerOptions {
     };
   };
 }
+
+declare interface WebSocketStreamOptions {
+  protocols?: string[];
+  signal?: AbortSignal;
+}
+
+declare interface WebSocketConnection {
+  readable: ReadableStream<string | Uint8Array>;
+  writable: WritableStream<string | Uint8Array>;
+  extensions: string;
+  protocol: string;
+}
+
+declare interface WebSocketCloseInfo {
+  code?: number;
```

Comment:
Closing with an empty object seems odd. Should this be an either-or-both type of thing?

Reply:
empty object/no value passed means default values.

Reply:
Okay, in that case I guess the `close` function should have a signature of `close(closeInfo?: WebSocketCloseInfo)` to allow omitting the empty object entirely.

Reply:
I guess so. The problem is the spec is barely non-existent, and the few things that exist are at times a bit contradictory (especially with the close stuff)